lib/utils.js

```diff
@@ -57,6 +57,12 @@ function parseCliFlags(flags) {
       currentFlag = 'js';
     }
 
+    if (flagValue === 'true') {
```

Comment:
I think this is treating the strings 'true' and 'false' as booleans no matter what flag is being set.
Am I reading this right?